scripts/qa_crowbarsetup.sh

```diff
@@ -1218,16 +1218,16 @@ function onadmin_set_source_variables
             CLOUDLOCALREPOS="SUSE-Cloud-5-devel"
         ;;
         develcloud6)
-            CLOUDSLE12DISTPATH=/ibs/Devel:/Cloud:/6/images/iso
+            CLOUDSLE12DISTPATH=${want_cloud6_iso_path:='/ibs/Devel:/Cloud:/6/images/iso'}
```

Comment:
This feels strange, as the default here is different than the default in line 1262 - this indicates that we should rather restructure how we do the mapping here (eg. read it with external tool from a yaml file that holds the mappings).
But as long as SAP only needs one variable here, this should not be a problem.